src/Features/Core/Portable/Completion/Providers/AbstractInternalsVisibleToCompletionProvider.cs

```diff
@@ -145,6 +150,9 @@ private static SyntaxNode GetAttributeSyntaxNodeOfToken(ISyntaxFactsService synt
             }
         }
 
+        private static bool IsProjectTypeBlacklisted(Project project)
+            => !project.SupportsCompilation;
```

Comment:
Is this going to break the ability to add references to F# projects which could potentially be supported?

Reply:
(I admit I don't know if F# projects set their assembly name properly, but if they do...we might want to support that.)

Reply:
P.P.S.: if they don't, it's better to have this change than to not have this change, but curious if we can do better still. :smile: